CRM/Core/Form/Task.php

```diff
@@ -177,11 +177,46 @@ public static function preProcessCommon(&$form) {
    * For example, for cases we need to override this function as the table name is civicrm_case_contact
    */
   public function setContactIDs() {
-    $this->_contactIds = CRM_Core_DAO::getContactIDsFromComponent($this->_entityIds,
+    $this->_contactIds = &getContactIDsFromComponent($this->_entityIds,
```

Comment:
is there a reason why we need the & here?

Reply:
Apologies, it was in the older version of the file - so no.